cookbooks/accounts/attributes/default.rb

```diff
@@ -4,3 +4,6 @@
 
 default[:accounts][:groups] = {}
 default[:accounts][:users] = {}
+
+# Needed in cookbooks/accounts/recipes/default.rb to assign group membership
+default[:etc] = {"passwd": {}}
```

Comment:
This is not really correct - the real problem is that you were using chef 15 which has the `Passwd` ohai plugin disabled by default. If we are ever able to switch to chef 15 then we would have to change that but for now the correct solution, which I have implemented on master, is to use chef 14 for the tests to match what we are running in production.

Reply:
👍 I see your change in 21187d302463ac14ce12ea0b42d27d277d2f175a and I’m running it locally right now to verify.

Reply:
Removed this in 5d218808c392f900121179e0d0c48c6ea5ce41b8 after testing.